src/localization/Ressources.cpp

```diff
@@ -0,0 +1,16 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman
+** File description:
+** Ressources
+*/
+
+#include "Ressources.hpp"
+
+namespace localization
+{
+    namespace Ressources
+    {
+        const RessourceString rsHello("Hello, World!");
```

Comment:
Change this to `constexpr`, once the constructor of RessourceString is constexpr